devs/docs/es-backports.rst

```diff
@@ -345,13 +345,13 @@ should be crossed out as well.
 - [ ] 952cf770ed6 Reestablish peer recovery after network errors (#57827)
 - [ ] 0987c0a5f3d Fix Broken Numeric Shard Generations in RepositoryData (#57813) (#57821)
 - [ ] 70e63a365a6 Refactor how to determine if a field is metafield (#57378) (#57771)
-- [ ] 004eb8bd7e4 Fix Bug With RepositoryData Caching (#57785) (#57800)
-- [ ] 5a4e5a1e9d3 Handle `cluster.max_shards_per_node` in YAML config (#57234)
-- [ ] 80d1b12fa3e Restore ThreadContext after Serializing OutboundMessage (#57659) (#57681)
-- [ ] fc4dd6d6813 Timeout health API on busy master (#57587)
-- [ ] e0a15e8dc4a Remove the 'array value parser' marker interface. (#57571) (#57622)
-- [ ] e50f5140921 IndexFieldData should hold the ValuesSourceType (#57373) (#57532)
-- [ ] ba2d70d8eb6 Serialize Outbound Messages on IO Threads (#56961) (#57080)
+- [x] 004eb8bd7e4 Fix Bug With RepositoryData Caching (#57785) (#57800)
+- [s] 5a4e5a1e9d3 Handle `cluster.max_shards_per_node` in YAML config (#57234)
+- [s] 80d1b12fa3e Restore ThreadContext after Serializing OutboundMessage (#57659) (#57681)
+- [x] fc4dd6d6813 Timeout health API on busy master (#57587)
+- [s] e0a15e8dc4a Remove the 'array value parser' marker interface. (#57571) (#57622)
+- [s] e50f5140921 IndexFieldData should hold the ValuesSourceType (#57373) (#57532)
+- [s] ba2d70d8eb6 Serialize Outbound Messages on IO Threads (#56961) (#57080)
```

Comment:
It produces issues and it's later on reverted (revert commit also marked as skipped)